site/cypress/e2e/Cypress-Gradeable/test_rubric_access.spec.js

```diff
@@ -0,0 +1,23 @@
+describe('Rubric Access Test', () => {
+    it('test sample file exists student', () => {
+        cy.login('aphacker');
+        cy.visit('/courses/s24/sample/download');
+        cy.get('.content').should('contain', "You don't have access to this page.");
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'details']);
+        cy.get('#error-0').should('exist').and('contain', 'You do not have permission to grade Grading Homework');
```

Comment:
you don't need the should('exist) here, and can make it just should('contain'... same as line 15, 18, and 20.

Reply:
I will approve and merge once this is changed.

Reply:
done ✅, i will take care of this from now thank u